lib/number_page.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+class NumberPage extends StatelessWidget {
+  const NumberPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const ListWidget();
+  }
+}
+
+class ListWidget extends StatelessWidget {
+  const ListWidget({super.key});
+  @override
+  Widget build(BuildContext context) {
+    var list = <Widget>[];
+    for (var i = 0; i <= 100; i++) {
+      list.add(const ListItemWidget());
```

Comment:
@RounakTadvi This piece of code creates 100 instances of ListItemWidget, each instance will hold the count value for itself and since we're using ListView.builder function we will lose the state of the widget when the widget goes out of view, to mitigate and fix this issue we'll need a variable to hold the values

Reply:
Resolved